{"additions": 38, "auther_ref": "pylint", "auther_sha": "23f645dd236928915c541233cdb8368ba13123b8", "author": "amaltaro", "body": "It's actually not working yet, but I can get back to it in the future.\r\n\r\nThe failure is quite weird, it complains about this path that does not exist (while building isort):\r\n```\r\n/build/amaltaro/w/slc7_amd64_gcc630/external/py2-backports.functools-lru-cache/1.5/etc/profile.d/init.sh: line 2: PY2_BACKPORTS.FUNCTOOLS_LRU_CACHE_ROOT=/build/amaltaro/w/slc7_amd64_gcc630/external/py2-backports.functools-lru-cache/1.5: No such file or directory\r\n```\r\n\r\nbut I can successfully list that directory\r\n```\r\namaltaro@vocms055:/build/amaltaro $ ls /build/amaltaro/w/slc7_amd64_gcc630/external/py2-backports.functools-lru-cache/1.5\r\netc  lib\r\n```\r\n\r\nNo clue of what happened yet...", "branch": "comp_gcc630", "changed_files": 10, "comments": 1, "commits": 1, "created_at": "1556097860", "deletions": 2, "labels": [], "number": 4904, "release-notes": [], "review_comments": 0, "state": "open", "title": "Build python2 pylint and its dependencies - DO NOT MERGE", "updated_at": "1556605587", "user": "amaltaro"}